test/index.spec.ts

```diff
@@ -81,46 +91,91 @@ describe('resin-semver', () => {
+				semver.compare('Resin OS 2.0.0+rev3', 'Resin OS 2.0.0+rev3.dev'),
+			).to.equal(1);
+			expect(
+				semver.compare('Resin OS 2.0.0+rev3.dev', 'Resin OS 2.0.0+rev3'),
+			).to.equal(-1);
+			expect(
+				semver.compare('Resin OS 2.0.0+rev3.dev', 'Resin OS 2.0.0+rev3.dev'),
+			).to.equal(0);
 			expect(semver.compare('2.0.0', 'Resin OS 2.0.0.dev')).to.equal(1);
 			expect(semver.compare('Resin OS 2.0.0.dev', '2.0.0')).to.equal(-1);
 			expect(semver.compare('Resin OS 2.0.0.dev', '2.0.0.dev')).to.equal(0);
 		});
 
 		it('should correctly compare "(dev)" and "(prod)" versions without revisions', () => {
-			expect(semver.compare('Resin OS 2.0.0', 'Resin OS 2.0.0 (dev)')).to.equal(1);
-			expect(semver.compare('Resin OS 2.0.0', 'Resin OS 2.0.0 (prod)')).to.equal(-1);
-			expect(semver.compare('Resin OS 2.0.0 (dev)', 'Resin OS 2.0.0 (prod)')).to.equal(-1);
-
-			expect(semver.compare('Resin OS 2.0.0', 'Resin OS 2.0.0-prod')).to.equal(1);
-			expect(semver.compare('Resin OS 2.0.0', 'Resin OS 2.0.0.prod')).to.equal(-1);
-			expect(semver.compare('Resin OS 2.0.0', 'Resin OS 2.0.0+prod')).to.equal(-1);
-			expect(semver.compare('Resin OS 2.0.0 (dev)', 'Resin OS 2.0.0+prod')).to.equal(-1);
-
-			expect(semver.compare('Resin OS 2.0.0.dev', 'Resin OS 2.0.0+dev')).to.equal(0);
-			expect(semver.compare('Resin OS 2.0.0.dev', 'Resin OS 2.0.0 (dev)')).to.equal(0);
-			expect(semver.compare('Resin OS 2.0.0+dev', 'Resin OS 2.0.0 (dev)')).to.equal(0);
-
-			expect(semver.compare('Resin OS 2.0.0.prod', 'Resin OS 2.0.0+prod')).to.equal(0);
-			expect(semver.compare('Resin OS 2.0.0.prod', 'Resin OS 2.0.0 (prod)')).to.equal(0);
-			expect(semver.compare('Resin OS 2.0.0+prod', 'Resin OS 2.0.0 (prod)')).to.equal(0);
+			expect(semver.compare('Resin OS 2.0.0', 'Resin OS 2.0.0 (dev)')).to.equal(
+				1,
+			);
+			expect(
+				semver.compare('Resin OS 2.0.0', 'Resin OS 2.0.0 (prod)'),
+			).to.equal(-1);
+			expect(
+				semver.compare('Resin OS 2.0.0 (dev)', 'Resin OS 2.0.0 (prod)'),
+			).to.equal(-1);
+
+			expect(semver.compare('Resin OS 2.0.0', 'Resin OS 2.0.0-prod')).to.equal(
+				1,
+			);
+			expect(semver.compare('Resin OS 2.0.0', 'Resin OS 2.0.0.prod')).to.equal(
+				-1,
+			);
+			expect(semver.compare('Resin OS 2.0.0', 'Resin OS 2.0.0+prod')).to.equal(
+				-1,
+			);
+			expect(
+				semver.compare('Resin OS 2.0.0 (dev)', 'Resin OS 2.0.0+prod'),
+			).to.equal(-1);
+
+			expect(
+				semver.compare('Resin OS 2.0.0.dev', 'Resin OS 2.0.0+dev'),
+			).to.equal(0);
+			expect(
+				semver.compare('Resin OS 2.0.0.dev', 'Resin OS 2.0.0 (dev)'),
+			).to.equal(0);
+			expect(
+				semver.compare('Resin OS 2.0.0+dev', 'Resin OS 2.0.0 (dev)'),
+			).to.equal(0);
+
+			expect(
+				semver.compare('Resin OS 2.0.0.prod', 'Resin OS 2.0.0+prod'),
+			).to.equal(0);
+			expect(
+				semver.compare('Resin OS 2.0.0.prod', 'Resin OS 2.0.0 (prod)'),
+			).to.equal(0);
+			expect(
+				semver.compare('Resin OS 2.0.0+prod', 'Resin OS 2.0.0 (prod)'),
+			).to.equal(0);
```

Comment:
I can see why this (and all the other changes in this file) have happened, and prettier is great, but this does make me a little sad. Could we boost the line length, so all these tests are readable (both here and in the docs), or is that not worth it?

Reply:
I'm happy to boost the line length here, readability is important for these test files